linode/manager
# fix: [M3-8686] - fix Buttons data-qa attributes

## Description 📝
Fixing a console error that has bothered me for a long time!

![Screenshot 2024-10-02 at 10 43 14](https://github.com/user-attachments/assets/11582835-9c61-4623-8677-3bb11087aafe)

## Changes  🔄
- fix `ActionPanel` `PrimaryButton` `data-qa` attribute in `SingleTextfieldForm`
- updates unit and e2e tests

## How to test 🧪

### Reproduction steps
- navigate to and reload `/profile/display`
- notice console error

### Verification steps
- navigate to and reload `/profile/display`
- confirm console error gone
- confirm unit and e2e suites are happy

## As an Author I have considered 🤔

*Check all that apply*
